src/app/services/dsm.service.ts

```diff
@@ -132,14 +132,14 @@ export class DSMService {
       if (json != null && json.filters != null) {
         viewFilterCopy = json.copy();
         for (let filter of json.filters) {
-          if (filter.type === Filter.OPTION_TYPE) {
+          if (filter.type === Filter.OPTION_TYPE && filter.participantColumn.tableAlias !== 'participantData') {
```

Comment:
did you test that change with other studies than just RGP and AT? What happens if type is option but not participantData?